src/lae_site/handlers/create_subscription.py

```diff
@@ -0,0 +1,169 @@
+from twisted.web.resource import Resource
+
+import traceback
+
+import attr
+
+from twisted.logger import Logger
+from twisted.web.server import NOT_DONE_YET
+
+from lae_util import stripe
+from lae_util.send_email import send_plain_email, FROM_ADDRESS
+
+from lae_site.handlers.web import env
+
+from lae_site.handlers.main import HandlerBase
+from lae_site.handlers.s4_signup_style import S4_SIGNUP_STYLE_COOKIE
+
+PLAN_ID                 = u'S4_consumer_iteration_2_beta1_2014-05-27'
+
+logger = Logger()
+
+
+class RenderErrorDetailsForBrowser(Exception):
+    def __init__(self, details):
+        self.details = details
+
+@attr.s
+class Stripe(object):
+    key = attr.ib()
+    # email = "vtsoneva@gmail.com"
+    def create(self, authorization_token, plan_id, email):
+        return stripe.Customer.create(
+            api_key=self.key,
+            card=authorization_token,
+            plan=plan_id,
+            email=email,
+        )
+
+class Mailer(object):
+    def mail(self, from_addr, to_addr, subject, headers):
+        return send_plain_email(
+            from_addr, to_addr, subject, headers,
+        )
+
+class CreateSubscription(HandlerBase):
+  def addHeaders(self, request):
+    request.setHeader('Access-Control-Allow-Origin', 'http://localhost:5000')
```

Comment:
this shouldnt be hardcoded as we will need different domains for different environments. @exarkun what would be your suggestion for most secure way of adding this to the server vars?

Reply:
this address is the client lektor site, and the header allows cross origin requests. So it will be different in staging and production environments